tests/console/check_system_info.pm

```diff
@@ -52,15 +52,33 @@ sub check_addons {
 }
 
 sub check_product {
-    my ($mypro) = @_;
-    $mypro //= uc get_var('SLE_PRODUCT', '');
+    my ($status) = @_;
+    $status //= "before";
     my $myver   = get_var('VERSION');
+    my %proname = (
+        HPC       => 'SLE-Product-HPC-' . $myver,
+        SLES      => 'SLES' . $myver,
+        SLED      => 'SLED' . $myver,
+        SLE_HPC   => 'SLE-Product-HPC-' . $myver,
+        leap      => "openSUSE-Leap",
+        Media_HPC => $myver . '-HPC',
+    );
+    my $mypro   = uc get_var('SLE_PRODUCT', '');
     my $product = '';
-    if ($mypro =~ /HPC/) {
-        $product = 'SLE-Product-HPC-' . $myver;
-    }
-    if ($mypro ne '') {
-        $product = $mypro . $myver;
+    if ($status eq 'before') {
+        if ((get_var("HDDVERSION") =~ /leap/)) {
+            $product = $proname{leap};
+        } elsif ($mypro eq "SLE_HPC") {
```

Comment:
So this means it is different for HPC product before migration and after migration? Verified on SLES15SP3? It maybe changed in the future I think.

Reply:
Not change in the future.  It is sles + hpc before migration, so product is sles. But after migraiton, it changes to hpc product.

Reply:
$mypro eq "SLE_HPC" means the base is for product SLE_HPC, why change the product name to SLES?  ($product = $proname{SLES};)